integrate with the latest vodml tools

pahjbo opened the pull request:
this should be harmless otherwise!

pahjbo committed:
initial vodml tooling setup

pahjbo committed:
quick example to show what generated XML schema are like

pahjbo committed:
updating for the latest tooling

pahjbo committed:
forgot this jar

lmichel commented:
Dear PAUL,

Your MR consists in adding gradle stuff to the repo. 
Can you be more explicit about the purpose of this by e.g. opening a Wiki page or an issue.
I'm also  surprised to see here and in all others DM projects duplicated Java code ; is that necessary?
Laurent

pahjbo commented:
The motivation behind doing this is get people to use the new tooling on all the models and to follow up on the long threads about VODML-IDs to show that most of the models so far do actually flow the patterns in appendix C of the standard document, as I have added that to the validation.

The gradle directory (+small jar) is the standard "bootstrap" that is done for gradle - its presence means that people do not have to install gradle separately - it can be removed if preferred, but then people have to install gradle separately.

mcdittmar commented:
I was kind-of excited to see this.. my first attempt to use gradle was stymied by java version, so didn't get far.  I'd like to move away from using Modelio for content, and just use it for diagrams, however.. I did picture gradle as an external tool so was surprised to seem more than just the connective files (ala vodml.properties for the xslt utilities).

pahjbo committed:
removed the gradle wrapper files

pahjbo committed:
removed the gradlew runner scripts

pahjbo commented:
I have removed the gradle wrapper scripts - they are not essential, but just to make life easier - it is necessary for you to install gradle manually now

lmichel commented:
We really need your tools to be promoted, but seeing them on a standard repo seems to be odd. I suggest you to put somewhere (README Woki?) a note telling what it is for and how to use it.

pahjbo commented:
There is the [Readme](https://github.com/ivoa/vo-dml/blob/master/tools/ReadMe.md) that explains the basic operation of the tools - however, I will write a note that also covers some more of the philosophy behind the new tooling that I have given in talks in the last few Interops.

Now all that this PR contains is the config files for running the tools - this is exactly equivalent to the config file [vo-dml/vo-dml.properties](https://github.com/ivoa-std/CoordinateDM/blob/master/vo-dml/vo-dml.properties). Though I think that this file is not much use in place for the ant tools as the relative links will be wrong, and the user will have had to manually download the tools repository. The idea of the gradle based stuff is that unless you are developing the tools themselves you do not need to download your own copy of from https://github.com/ivoa/vo-dml (as was previously necessary with the ant based tools), but can work in place in each of the DM standard repos.

pahjbo committed:
add task to generate vodml from the XMI

pahjbo committed:
using autogenerated dependency vodsl

pahjbo committed:
update unfortunate description

pahjbo committed:
update to latest vodml plugin

pahjbo committed:
update to latest vodml plugin

pahjbo committed:
Merge branch 'master' into vodmlplug

pahjbo committed:
remove vodsl from this branch

This branch is supposed to only represent adopting the new tools for existing model
